build-parent/pom.xml

```diff
@@ -83,6 +83,13 @@
         <opensearch.image>opensearchproject/opensearch:${opensearch-server.version}</opensearch.image>
         <opensearch.protocol>http</opensearch.protocol>
 
+        <!-- Database images for JDBC/Reactive/Hibernate tests -->
+        <postgres.image>postgres:14.1</postgres.image>
+        <mariadb.image>mariadb:10.4</mariadb.image>
```

Comment:
We had some issues with Podman default config requiring a registry to be defined so maybe it's a good opportunity to change the Docker Hub ones to `docker.io/...`.